Automattic/wp-calypso
# Nav redesign: Show masterbar and tidy up global sidebar

Fixes https://github.com/Automattic/dotcom-forge/issues/8040
Fixes https://github.com/Automattic/dotcom-forge/issues/8150
Fixes https://github.com/Automattic/dotcom-forge/issues/8151
Fixes https://github.com/Automattic/dotcom-forge/issues/8152

Combining PRs here to make it simpler to test and deploy.

https://github.com/Automattic/wp-calypso/assets/5560595/9a2264b3-f348-43f0-897d-6959fc44a241


## Proposed Changes

* This updates the layout to render the masterbar on the global pages (All sites, Reader and Me/Profile)
* This also updates the styles to move the pages content to make room for the masterbar
* This hides the global sidebar header & footer that should be no longer needed when we are showing the masterbar on all pages
* Removes back links from Reader and Me pages

## Why are these changes being made?


* Tidying up UX

## Testing Instructions



* Go to calypso.live and confirm masterbar loads and behaves on all global pages
* Confirm the sidebar works and is styled correctly at different dimensions

## Pre-merge Checklist
